packages/elements/src/components/mutant-view/mutant-view.ts

```diff
@@ -118,7 +118,7 @@ const COLUMNS: Column<Metrics>[] = [
   {
     key: 'totalMutants',
     label: 'Total',
-    tooltip: 'All mutants (valid + invalid + ignored)',
+    tooltip: 'All mutants (valid + invalid + ignored + pending)',
```

Comment:
I would say pending is already included in valid. Though I'm not sure why we mention valid + invalid + ignored since valid and invalid are not columns in this table.

```suggestion
    tooltip: 'All mutants (except runtimeErrors+ compileErrors)',
```